common/src/main/java/org/opensearch/ml/common/MLModelGroup.java

```diff
@@ -203,6 +203,7 @@ public static MLModelGroup parse(XContentParser parser) throws IOException {
                     break;
                 case TENANT_ID:
                     tenantId = parser.textOrNull();
+                    break;
```

Comment:
Do we need to rebase? I thought we made these changes in another PR already?

Reply:
No need to rebase as it’ll be a no op.  I didn’t want to kill passing tests on a rebase.